lib/analyzer/apk-analyzer.ts

```diff
@@ -1,19 +1,30 @@
 import { Docker, DockerOptions } from "../docker";
+import { DEFAULT_ENCODING } from "./image-extractor";
 import { AnalyzerPkg } from "./types";
```

Comment:
Making the encoding abstracted here is probably a little bit much, I think it's fine to inline "utf8" in the function call; fairly standard stuff.

Reply:
Done